ui/src/CollectionExerciseDetails.js

```diff
@@ -443,6 +533,17 @@ class CollectionExerciseDetails extends Component {
           <TableCell component="th" scope="row">
             {actionRule.packCode}
           </TableCell>
+          <TableCell component="th" scope="row">
+            {actionRule.hasTriggered ? null : (
```

Comment:
Correct me if I've missed this in the code, but I believe this button might have a buggy state where it'll still show if the user doesn't have the required permission, but fail when they try to submit the change? It's hard to test locally.

Reply:
That would be correct it brings up an alert with a 403 forbidden error, there are ways around it but I'm not sure what would be the best;
I could make it so it checks before it adds the button if the user has the correct perms, but it would require a GET request for every action rule - since there is no perm for editing an action rule but there are rules for creating each specific type of action rule or I could have it so when someone presses the button, before the dialog appears it would check the perms of the user - which I think is the best way to do it?